README.md

```diff
@@ -14,6 +14,7 @@ A collection of new grad roles for 2020.
 | Example | Site Reliability Engineer | San Francisco, CA, USA | https://example.com/job |
 | Akuna Capital | Junior Developer | Chicago, IL | https://akunacapital.com/job-details?gh_jid=1755321 |
 | American Express | Software Engineer | Phoenix, AZ, USA; Ft. Lauderdale, FL, USA; New York, NY, USA | https://jobs.americanexpress.com/jobs/19013265?lang=en-us&previousLocale=en-US |
+| Cisco | Software Engineer | Multiple Locations | https://jobs.cisco.com/jobs/ProjectDetail/Software-Engineer-Bachelor-s-Full-Time-United-States/1265288?source=Cisco+Jobs+Career+Site&tags=CDC+SnNG+engineering-university-program | 
```

Comment:
Could you please list the locations? Makes it easier for people to CTRL+F The page

Reply:
Got you